diagrams/request_uri_mode_create.md

```diff
@@ -0,0 +1,49 @@
+```plantuml
+@startuml
+
+autonumber
+
+box "Wallet"
+participant "Metadata" as wm
+participant "Authorization Endpoint" as w
+end box
+
+participant "User Agent" as u
+
+box "Verifier"
+participant "Frontend" as r
+participant "Request Endpoint" as rp
+participant "Response Endpoint" as rb
+end box
+
+u --> r : use
+activate r
+
+r --> u: authorization request\n(client_id, request{request_uri, request_uri_mode=POST, state})
+deactivate r
+u --> w: authorization request\n(client_id, request{request_uri, request_uri_mode=POST, state})
+activate w
+w --> w: check authorization request signature
+w --> w: check on trustworthiness of Verifier (approach on trust mechanism)
```

Comment:
```suggestion
w --> w: evaluate the reliability of the Verifier (trust evaluation).
```

Reply:
reliability is not the right association with trust evaluation imo

Reply:
I think trust (or trustworthiness) is the right term. If there is no trust, the Wallet should not send a request to the Verifier's request URI.